### PR TITLE
mesa 22.1.7

### PR DESCRIPTION
**Pull request by yurikoles**
- mesa 22.1.7
- mesa: update Pygments resource to 2.13.0

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by yurikoles** [comment]
Please add label `CI-no-fail-fast`.

A lot of unrelated failures with Qt / qmake on Linux: `Project ERROR: Cannot run compiler 'g++-11'. Output:`

**Comment by yurikoles** [comment]
Whoops, my `qt*` revision bumps are unneeded, they should be fixed by #107400

File: Formula/guichan.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Guichan < Formula
   end
 
   resource "fixedfont.bmp" do
-    url "https://guichan.sourceforge.io/oldsite/images/fixedfont.bmp"
+    url "http://guichan.sourceforge.io/oldsite/images/fixedfont.bmp"
     sha256 "fc6144c8fefa27c207560820450abb41378c705a0655f536ce33e44a5332c5cc"
   end
```
Comment (carlocab) [inline Formula/guichan.rb:30]: Do we need this change in your PR?

Comment (yurikoles) [reply Formula/guichan.rb:30]: I see a redirect to http in a browser and curl SSL error on the Linux runner.

Comment (carlocab) [reply Formula/guichan.rb:30]: We can do it in a separate PR so we don't have to republish this formula's bottles.

Comment (yurikoles) [reply Formula/guichan.rb:30]: Here you go: #108581

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**